### ISSUE TITLE
[Suggestion] 

### ISSUE THREAD

**Issue by 2Naato** [TeamTwilight/twilightforest#1730]
### The Suggestion : A config for the final boss

Would it be possible to **include a config that allows to specify which mob spawns instead of the "final boss"** (so in that purple box at the top of the white castle). 
As of now I believe that there's still a buffed goblin as a placeholder or something (or maybe just nothing). Personally I would prefer for the specified mob to spawn only when a player enters the purple cube (It was the case for the goblin if I remember well...).
**_So put simply : I would appreciate having a config option where one writes "mod_id:mob_id" and voila, that's the final boss !_** 

### Why Should It Be in the Mod?

Not having a final goal just doesn't quite make it for me, so I thought that it could be a good idea to have a (boss) mob from another mod as the final boss !

### Please Read and Confirm the Following

- [X] I have confirmed this suggestion is unique and has not been suggested already.

**Comment by GizmoTheMoonPig** [comment]
this is not something that will be going on the base mod, however one of our developers created an addon that allows you to do just what youre looking for. You can get it here: https://www.curseforge.com/minecraft/mc-mods/twilight-tweaks
